src/utils/object.js

```diff
@@ -1,13 +1,24 @@
 import { from } from './list';
 
-export const shallowMerge = (...objects) => {
+export const merge = (...objects) => {
+
+  const isObject = (obj) => {
+    return obj && typeof obj === 'object';
+  };
+
   return objects.reduce((x, y) => {
```

Comment:
I think it would be nice to update this to use more descriptive names for readability. maybe `result` for `x` and `source` for `y`